cirrus/server/tests/test_main.py

```diff
@@ -99,8 +99,9 @@ def test_get_features_v2_with_required_field(client):
             },
             status.HTTP_400_BAD_REQUEST,
             "Client ID value is missing or empty",
+            id="empty_client_id",
```

Comment:
I went through all the parametrized tests and added ids because i couldn't tell them apart when debugging